src/01-Learning-JS/08-load-data.md

```diff
@@ -234,12 +242,13 @@ In this case, load the following CSV file: `nc_absentee_mail_2024.csv`.
 2. In the second codeblock, use `console.log()` to log the following properties to the console: `.name` and `.size`.
 3. Still in the second codeblock, add a second `console.log()` and log the first object in the Array to the console.
 
-```javascript
-// Your FileAttachment() code goes here
+```js
+const nc_Absentee_Mail = FileAttachment("./../data/nc-voters/nc_absentee_mail_2024.csv").csv({typed: true})
 ```
 
-```javascript
-// Your other code with your variable goes here
+```js
+console.log(nc_Absentee_Mail.name)
```

Comment:
Ah, I made an error on this assignment by not guiding you through a process to log the correct information. The .name and .size won't be available, once you encode the attached data with `.csv()` at the end of the `FileAttachment()` method.

My bad! I will revise this assignment with clearer guidance here.